src/main/java/com/aws/greengrass/mqtt/bridge/clients/LocalMqtt5Client.java

```diff
@@ -151,6 +157,8 @@ public class LocalMqtt5Client implements MessageClient<MqttMessage> {
         this.add(SubAckPacket.SubAckReasonCode.WILDCARD_SUBSCRIPTIONS_NOT_SUPPORTED.getValue());
     }};
     private Future<?> updateSubscriptionsTask;
+    @Getter
+    private PublishPacket lastPublish;
```

Comment:
remove this. You do not need this to verify the functionality. You have a full broker and a full MQTT client running.